index.js

```diff
@@ -0,0 +1,93 @@
+const core = require("@actions/core");
+const exec = require("@actions/exec");
+const common = require("@zaproxy/actions-common-scans");
+const _ = require("lodash");
+
+// Default file names
+let mdReportName = "report_md.md";
+let htmlReportName = "report_html.html";
```

Comment:
No jsonReportName? The other 2 actions support this

Reply:
hi @psiinon, I've updated the code with support jsonReportName like the other 2 actions 😄